tools/WLED_ESP32_2MB_C3.csv

```diff
@@ -0,0 +1,5 @@
+# Name,   Type, SubType, Offset,  Size, Flags
+nvs,      data, nvs,     0x9000,    20K,
+otadata,  data, ota,     0xe000,    8K,
+app0,     app,  ota_0,   0x10000,   1536K,
+spiffs,   data, spiffs,  0x190000,  384K,
```

Comment:
just a small file renaming here - to be more consistent with other partition tables - 
please name this file `WLED_ESP32-C3_2MB.csv`